src/DevBetterWeb.Infrastructure/Handlers/DailyCheckInitiatedEventHandler.cs

```diff
@@ -46,12 +53,31 @@ public async Task Handle(DailyCheckInitiatedEvent domainEvent)
     // check if number of MemberSubscriptionPlans == expected number
     await _dailyCheckSubscriptionPlanCountService.WarnIfNumberOfMemberSubscriptionPlansDifferentThanExpected(messages);
 
+    await UpdateVideosThumbnail();
+
     messages.Append(DAILY_CHECK_COMPLETED_MESSAGE);
 
     await SendMessagesToDiscord(messages);
     await StoreMessagesInTasksCompleted(messages);
   }
 
+  private async Task UpdateVideosThumbnail()
```

Comment:
Pass messages like the other methods do, and if you do anything (like add a thumbnail) note that by appending to the message collection.

Reply:
Done